example-get-started/code/params.yaml

```diff
@@ -2,10 +2,11 @@ prepare:
   split: 0.20
   seed: 20170428
 
-featurize:
-  max_features: 500
+fr:
```

Comment:
@dmpetrov @dberenbaum I know where it comes from, but don't you think it's better for the entry tutorial to be extra explicit and friendly? Does it hurt the layout too much with the longer names?

(eventually we'll need to figure out the UI to cover longer names anyway, right?)

Reply:
I knew I should have kept this separate 😆! I mentioned it to @dmpetrov and looked through the docs to see if it seemed hard to follow the parameters with the short names. I don't think it's too bad, but I can see both sides of it.

Reply:
> I knew I should have kept this separate

yeah, since it'll force us to go and update docs in a lot of places (besides this discussion itself) :)

Reply:
I'm totally fine if you discussed and agreed that it makes sense. Up to you Dave on this.

Reply:
I went with a compromise. The only place I notice a difference visually with the shorter names is with the train parameters that make `dvc exp show` output too wide, so only those parameters are shortened:

* `n_estimators` -> `n_est`
* `min_samples_split` -> `min_split`

This keeps the table width reasonable in the example without too much loss of clarity.